wrapper/src/main/resources/messages.properties

```diff
@@ -214,6 +214,7 @@ AuroraStaleDnsHelper.reset=Reset stored writer host.
 OpenedConnectionTracker.unableToPopulateOpenedConnectionQueue="[Internal Error] The driver is unable to track this opened connection because the instance endpoint is unknown."
 OpenedConnectionTracker.invalidatingConnections="Invalidating opened connections to host: ''{0}''"
 
+DatabaseDialect.unsupportedDriverProtocol=Driver protocol ''{0}'' is not supported.
```

Comment:
Can you please remove "ReadWriteSplittingPlugin.unsupportedDriverProtocol=Driver protocol ''{0}'' is not supported." from this file as well? Seems like we are no longer using it.